app/lib/features/chat/pages/chat_page.dart

```diff
@@ -19,7 +19,9 @@ class ChatPage extends ConsumerWidget {
   @override
   Widget build(BuildContext context, WidgetRef ref) {
     final client = ref.watch(clientProvider)!;
+    final chatsNotifier = ref.read(chatListProvider.notifier);
```

Comment:
again ```ref.watch()```. It's more recommended and has no side effects.

Reply:
Fixed by https://github.com/acterglobal/a3/pull/855/commits/cb76448736e09d50aeb157ab7e7a27aecbd219d6